test/NuGet.Core.FuncTests/NuGet.Commands.FuncTest/RestoreCommandTests.cs

```diff
@@ -1956,8 +1963,7 @@ public async Task RestoreCommand_RestoreFloatingVersionWithFailingHttpSourceAsyn
             // Arrange
             var sources = new List<PackageSource>
             {
-                new PackageSource("https://failingSource"),
-                new PackageSource(NuGetConstants.V3FeedUrl)
```

Comment:
Why was this source removed?

Reply:
transient bad state made me do it. i shouldn't have done it. put it back now...and things are failing as they should.
fixed in next iteration.